cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,33 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io/#/login');
+  });
+
   it('should provide an ability to log in', () => {
+    const userName = `matesobaka@mate.ua`;
+    const Password = `123123123`;
+
+    cy.url()
+      .should('include', '/login');
+    cy.get('h1.text-xs-center')
+      .should(`be.visible`);
+
+    cy.get('h1.text-xs-center')
+      .should('contain.text', 'Sign In');
+    cy.get(':nth-child(1) > .form-control')
+      .type(userName);
```

Comment:
Change the name of the variable userName to email, since the email address is used for login.